mmv1/third_party/terraform/tests/resource_bigtable_instance_test.go

```diff
@@ -209,8 +209,10 @@ func TestAccBigtableInstance_createWithAutoscalingAndUpdate(t *testing.T) {
 			{
 				// Create Autoscaling config with 2 nodes.
 				Config: testAccBigtableInstance_autoscalingCluster(instanceName, 2, 5, 70),
-				Check: resource.ComposeTestCheckFunc(resource.TestCheckResourceAttr("google_bigtable_instance.instance",
-					"cluster.0.num_nodes", "2")),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr("google_bigtable_instance.instance", "cluster.0.num_nodes", "2"),
+					resource.TestCheckResourceAttr("google_bigtable_instance.instance", "cluster.0.autoscaling_config.0.storage_target", "8192"),
```

Comment:
Can you add a comment and say something like: 8192 is the expected default storage utilization limit per HDD node?

Reply:
Yes.

Reply:
This seems dangerous to check explicitly. It has us replying on default api behavior. If api changes this default at any point this test will fail. Can we check value set to Int rather then checking specific value?

Reply:
On the contrary I think the test _should_ fail to catch this if the API is broken and not setting the default value properly? I find it more assuring that this is checking the e2e result at this detailed level. Similarly, our gcloud e2e tests also check specific behaviors of the API.

Reply:
I agree with Weihan. I don't think this is dangerous. The dangerous thing in storage are things like silent data corruption, data loss, etc. Failing a test is not one of the dangerous things I can think of. In fact, fail to catch default value change could let unexpected change go through.

Reply:
ACK, not a blocker